doc/usage/tutorial.md

```diff
@@ -162,3 +208,125 @@ NOTE:
+E0327 06:30:45.496355       1 reflector.go:134] github.com/kubernetes-sigs/kube-batch/pkg/scheduler/cache/cache.go:308: Failed to list *v1alpha1.Queue: queues.scheduling.incubator.k8s.io is forbidden: User "system:serviceaccount:kube-system:default" cannot list queues.scheduling.incubator.k8s.io at the cluster scope
+E0327 06:30:45.497695       1 reflector.go:134] github.com/kubernetes-sigs/kube-batch/pkg/scheduler/cache/cache.go:301: Failed to list *v1beta1.PodDisruptionBudget: poddisruptionbudgets.policy is forbidden: User "system:serviceaccount:kube-system:default" cannot list poddisruptionbudgets.policy at the cluster scope
+E0327 06:30:45.498641       1 reflector.go:134] github.com/kubernetes-sigs/kube-batch/pkg/scheduler/cache/cache.go:303: Failed to list *v1.Node: nodes is forbidden: User "system:serviceaccount:kube-system:default" cannot list nodes at the cluster scope
+E0327 06:30:45.499895       1 reflector.go:134] github.com/kubernetes-sigs/kube-batch/pkg/scheduler/cache/cache.go:305: Failed to list *v1.PersistentVolume: persistentvolumes is forbidden: User "system:serviceaccount:kube-system:default" cannot list persistentvolumes at the cluster scope
+
+```
+
+`kube-batch` need to collect cluster information(such as Pod, Node, CRD, etc) for scheduling, so the service account used by the deployment must have permission to access those cluster resources, otherwise, `kube-batch` will fail to startup. For users who are not familiar with Kubernetes RBAC, please copy the example/role.yaml into `$GOPATH/src/github.com/kubernetes-sigs/kube-batch/deployment/kube-batch/templates/` and reinstall batch.
+
+
+### How to enable gang-scheduling on kubeflow/tf-operator for kube-batch?
+Please follow tf-operator [quick start](https://www.kubeflow.org/docs/components/tftraining/#quick-start) to install tf-operator.
+
+1. Add following rules to `tf-job-operator` clusterrole to grant tf-job-operator permission to create `PodGroup`.
+
+```yaml
+- apiGroups:
+  - scheduling.incubator.k8s.io
+  resources:
+  - podgroups
+  verbs:
+  - '*'
+
+```
+
+2. Enable gang-scheduling in `tf-operator` by add flag `--enable-gang-scheduling=true`
+```
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: tf-job-operator
+  ....
+spec:
+.....
+    spec:
+      containers:
+      - command:
+        - /opt/kubeflow/tf-operator.v1beta2
+        - --alsologtostderr
+        - -v=1
+        - --enable-gang-scheduling=true
+```
+
+3. Try to submit following training jobs. If you have less than 2 gpus available in your cluster, none of workers will be scheduled.
+After `tf-operator` v0.5, `tf-operator` will create `PodGroup` and use it for gang-scheduling rather than PDB which will be deprecated.
+In order to guide `tf-operator` to use `PodGroup`, you need to annotate `scheduling.k8s.io/group-name: ${PodGroupName}` to your job. In this case,
+`tf-operator` will create a `PodGroup` named `gang-scheduling-podgroup`.
+
+```yaml
+apiVersion: "kubeflow.org/v1beta1"
```

Comment:
Hi @Jeffwan can you provide kubeflow version information. I can't run this example in kubeflow v0.5.0-rc.1.

And previously example [gang-scheduling](https://www.kubeflow.org/docs/other-guides/job-scheduling/) are support `PodGroup`?

Reply:
kubeflow v0.5.0-rc1 should be fine. PodGroup support was merged into TF Operator and then community cut 0.5. I normally just use kubeflow@master. Since 0.5 was cut last week, I didn't see extra changes there. 

How do you setup your environment? What I did is create a cluster with 2 gpus, create a busy box which takes 1 gpu. I schedule this TFJob. At this moment. cluster only have 1 gpu available and kube-batch will not schedule this tfJob. 

Then I delete busy box and release 1 gpu. TFJob get scheduled successfully.

Reply:
Let me know if you meet other problems. You can also look at kube-batch logs. Ping me on slack - Jiaxin Shan.

Reply:
@Jeffwan Thanks, If I have other problems will share on Slack.